src/helm/helmfile.yaml

```diff
@@ -0,0 +1,27 @@
+repositories:
```

Comment:
You do not use external repositories so maybe it's better to remove this part.

Reply:
Done, thanks